src/spikeinterface/exporters/to_ibl.py

```diff
@@ -0,0 +1,280 @@
+from __future__ import annotations
+
+from importlib.util import find_spec
+import os
+import shutil
+import warnings
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+from tqdm.auto import tqdm
+
+from spikeinterface.core import ChannelSparsity, SortingAnalyzer
+from spikeinterface.core.job_tools import divide_segment_into_chunks
+from spikeinterface.core.template_tools import get_template_extremum_channel
+from spikeinterface.exporters import (
+    export_to_phy,
+)
+
+
+def export_to_ibl(
+    analyzer: SortingAnalyzer,
+    output_folder: str | Path,
+    lfp_recording=None,
+    rms_win_length_sec=3,
+    welch_win_length_samples=2**14,
+    total_secs_spec_dens=100,
+    only_ibl_specific_steps=False,
+    sparsity: Optional[ChannelSparsity] = None,
+    remove_if_exists: bool = False,
+    verbose: bool = True,
+    **job_kwargs,
+):
+    """
+    Exports a sorting analyzer to the IBL gui format (similar to the Phy format with some extras).
+
+    Parameters
+    ----------
+    analyzer: SortingAnalyzer
+        The sorting analyzer object to use for spike information.
+        Should also contain the pre-processed recording to use for AP-band data.
+    output_folder: str | Path
+        The output folder for the exports.
+    lfp_recording: Any SI Recording object, default None
+        The pre-processed recording to use for LFP data. If None, the LFP data is not exported.
+    rms_win_length_sec: float, default: 3
+        The window length in seconds for the RMS calculation (on the LFP data).
+    welch_win_length_samples: int, default: 2^14
+        The window length in samples for the Welch spectral density computation (on the LFP data).
+    total_secs_spec_dens: int, default: 100
+        The total number of seconds to use for the spectral density calculation.
+    only_ibl_specific_steps: bool, default: False
+        If True, only the IBL specific steps are run (i.e. skips calling `export_to_phy`)
+    sparsity: ChannelSparsity or None, default: None
+        The sparsity object (currently only respected for phy part of the export)
+    remove_if_exists: bool, default: False
+        If True and "output_folder" exists, it is removed and overwritten
+    verbose: bool, default: True
+        If True, output is verbose
+
+    """
+
+    if find_spec("scipy") is None:
+        raise ImportError("Please install scipy to use the export_to_ibl function.")
```

Comment:
```suggestion
        raise ImportError("Please install scipy to use the `export_to_ibl` function.")
```

just our convention to put ticks around library things (so functions or args). Since we just started this convention not everything is done this way, but with stuff moving forward we want to enforce it :)